### PR TITLE
(WIP) : Update image caption

### PR DESCRIPTION
**Pull request by eostrovski**
## To Do:
Summarize changes, ask for reviews , create task list, ask questions

## For Review
Please review

### CODE REVIEW

**Comment by eostrovski** [comment]
Comment

File: _posts/0000-01-07-eostrovski.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Construction cat with yellow hat
```
Comment (patelgp) [inline _posts/0000-01-07-eostrovski.md]: Caption is ok

**Review by patelgp — CHANGES_REQUESTED** [review]
Need to change caption

**Review by patelgp — APPROVED** [review]

**Comment by eostrovski** [comment]
Merging pull request

**Pull request merged**